devcodeabode/MoDiBo
# Remove GNU Make Dependency

**Related bugs/feature requests**



+ Resolves  #21 

**Describe the changes**

Moves commands from a Makefile into the npm scripts themselves, removing the GNU Make dependency

**Expected behavior**

The same behavior should occur as before.

**Screenshots**


**New npm dependencies**



**Testing Instructions**


Verify that each of the scripts that previously relied on GNU Make still works without a Makefile

- [ ] `npm run update`
- [ ] `npm run clean`
- [ ] `npm run cleanLogs`

**Additional context**
